pact4s/src/test/scala/org/broadinstitute/dsde/workbench/sam/MockTestSupport.scala

```diff
@@ -36,37 +36,37 @@ import scalikejdbc.withSQL
 import java.util.concurrent.Executors
 import scala.concurrent.ExecutionContext.Implicits.{global => globalEc}
 import scala.concurrent.duration._
-import scala.concurrent.{Await, Awaitable, ExecutionContext}
+import scala.concurrent.{Await, Awaitable, ExecutionContext, ExecutionContextExecutor}
 
 trait MockTestSupport {
   def runAndWait[T](f: Awaitable[T]): T = Await.result(f, Duration.Inf)
   def runAndWait[T](f: IO[T]): T = f.unsafeRunSync()
 
-  implicit val futureTimeout = Timeout(Span(10, Seconds))
+  implicit val futureTimeout: Timeout = Timeout(Span(10, Seconds))
```

Comment:
Just curious - is adding all these type definitions necessary for the changes made in this PR, or just cleaning up?

Reply:
These types are just for cleaning to eliminate `IntelliJ` warnings.